rapidfire.gemspec

```diff
@@ -22,6 +22,7 @@ PIM
 
   s.add_dependency 'rails', '>= 3.2.13'
   s.add_dependency 'active_model_serializers', '~> 0.8.1'
+  s.add_dependency 'kaminari'
```

Comment:
Can you make this a soft dependency? In code check if `Kaminari` is defined, and then do pagination.

Reply:
Yeah, will do that.